lib/src/components/NxIconDropdown/NxIconDropdown.scss

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@import '../../scss-shared/nx-text-helpers';
+@import '../../scss-shared/nx-container-helpers';
+@import '../../scss-shared/nx-button-helpers';
+@import '../../scss-shared/nx-dropdown-variables';
+
+.nx-icon-dropdown {
+  width: initial;
+
+  .nx-dropdown-menu {
+    right: 0;
+  }
+}
+
+.nx-icon-dropdown__toggle {
+  margin-right: 0;
+
+  &.open {
+    border-color: var(--nx-swatch-blue-40);
```

Comment:
This matches `NxDropdown`'s current impl but as we discussed in one of the kaleido meetings this aspect of `NxDropdown` is not correct, and also this doesn't match the `NxIconDropdown` mockups.

Reply:
> This matches `NxDropdown`'s current impl but as we discussed in one of the kaleido meetings this aspect of `NxDropdown` is not correct, and also this doesn't match the `NxIconDropdown` mockups.

Fixed here per Sketch: https://github.com/sonatype/sonatype-react-shared-components/pull/465/commits/19c494acd418faef5fd053ba1cb9c39332ad06ee

Reply:
> Fixed here per Sketch: 19c494a

The background color is missing

Reply:
> > Fixed here per Sketch: [19c494a](https://github.com/sonatype/sonatype-react-shared-components/commit/19c494acd418faef5fd053ba1cb9c39332ad06ee)
> 
> The background color is missing

Fixed here: https://github.com/sonatype/sonatype-react-shared-components/pull/465/commits/c7554c01b9ca273df678bc1c574e8eebee1dc3cb